qucs/qucs/diagrams/marker.cpp

```diff
@@ -363,18 +496,24 @@ void Marker::paint(ViewPainter *p, int x0, int y0)
   x2_ = p->drawText(Text, x0+x1+3, y0+y1+3, &y2_);
   x2_ += int(6.0*p->Scale);
   y2_ += int(6.0*p->Scale);
-  if(!transparent) {
```

Comment:
why the logic of 'transparent' was changed? Previously the marker border was always drawn and 'transparent' referred to the rectangle area, which was not transparent by default.
Now selecting 'transparent' removes the border and makes the rectangle area non-transparent.